RELEASE.txt

```diff
@@ -17,10 +17,10 @@ NOTE: If you build with Python 3.10.0 and then rebuild with 3.10.1 (or higher),
       see unexpected rebuilds. This is due to Python internals changing which changed
       the signature of a Python Action Function.
 
-NOTE: If you use a dictionary to specify your CPPDEFINES, the order of
-      dictionary keys is now preserved when generating the command line.
-      Previously these were sorted alphabecially: this may cause a change
-      which leads to a rebuild.
+NOTE: If you use a dictionary to specify your CPPDEFINES, you may see unexpected build results
```

Comment:
"unexpected" seems a bit strong when combined with "results".  Unless the possibly changed order affects the actual build, which is really unlikely, it's a single rebuild (and silently, update of sconsign), but the *result* will be the same.

Reply:
If you update SCons to 4.5.0 and you didn't change any sources, and your project rebuilds. I'd call that unexpected.
This has long been the practice of SCons to warn about such, becuase we almost always will get questions about such if we don't note them.

Reply:
A rebuild may be unexpected, and this is warning about it.  But "unexpected...results" suggests the build may not end up the same, which sounds worse than a rebuild to me.  How about this:

> NOTE: If you use a dictionary to specify all or part of CPPDEFINES, you may
      see one-time rebuilds due to changed order of defines on the command 
      line. SCons now retains the order of addition of *all* entries to
      CPPDEFINES. In previous versions a dictionary used for addition was
      sorted alphabetically before the individual items were added.

Reply:
change to `unexpected rebuild` ?